tracee/event_monitor_ebpf.c

```diff
@@ -1466,8 +1467,15 @@ int trace_security_bprm_check(struct pt_regs *ctx, struct linux_binprm *bprm)
     context.argnum = 1;
     context.retval = 0;
 
+    int idx = 0;
+    // Get per-cpu string buffer
+    submit_buf_t *string_p = string_buf.lookup(&idx);
+    if (string_p == NULL)
+        return -1;
+    get_path_string(string_p, &bprm->file->f_path);
+
     save_context_to_buf(submit_p, (void*)&context);
-    save_path_to_buf(submit_p, &bprm->file->f_path);
+    save_str_to_buf(submit_p, (void *)&string_p->buf[string_p->off], STR_T);
```

Comment:
does `save_str_to_buf` still need the `type` arg?

Reply:
Good catch. I'll remove this